File: fix_syntax.py
with open('frontend/src/pages/Cases.tsx', 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i in range(len(lines)):
    if "= /review/\\; }}" in lines[i]:
        lines[i] = lines[i].replace("= /review/; }}", "")

with open('frontend/src/pages/Cases.tsx', 'w', encoding='utf-8') as f:
    f.writelines(lines)
print("Syntax fixed")

File: fix_syntax.js
const fs = require('fs');
const file = 'frontend/src/pages/crew/DigitalPRFForm.tsx';
let content = fs.readFileSync(file, 'utf8');

content = content.replace("id: \\`crew\\${i + 3}\\`, tag: \\`Crew \\${i + 3}\\`,", "id: `crew${i + 3}`, tag: `Crew ${i + 3}`,");

fs.writeFileSync(file, content);
console.log('Fixed syntax error!');
